README.md

```diff
@@ -298,7 +298,7 @@ API | Description | Auth | HTTPS | Link |
 | Clash Royale | Clash Royale Game Information | No | Yes | [Go!](https://github.com/martincarrera/clash-royale-api) |
 | Comic Vine | Comics | No | Yes | [Go!](https://comicvine.gamespot.com/api/documentation) |
 | Deck of Cards | Deck of Cards | No | No | [Go!](http://deckofcardsapi.com/) |
-| Destiny The Game | Bungie Platform API | `apiKey` | Yes | [Go!](https://www.bungie.net/en/Clan/Post/39966/85087279/0/0) |
+| Destiny The Game | Bungie Platform API | `apiKey` | Yes | [Go!](https://github.com/Bungie-net/api) |
```

Comment:
Why are you suggesting to change the link? I was able to follow the docs on the original link and access a few APIs from `bungie.net` rather than the `github.io` link.

Reply:
Hi. The old API link is deprecated. See https://www.bungie.net/en/Groups/Post?groupId=39966&postId=230625723&sort=0&

The github.io link is the new api home. This is due to the release of Destiny 2. 
page=0